LEGO1/lego/legoomni/src/input/legoinputmanager.cpp

```diff
@@ -191,8 +189,13 @@ MxResult LegoInputManager::GetJoystickState(MxU32* p_joystickX, MxU32* p_joystic
 			return FAILURE;
 		}
 
+#ifdef WINDOWS_STORE
```

Comment:
we should not be using platform specific code in LEGO1.  If SDL doesn't abstract this properly for us (is that expected?) we need to add our own interface abstraction

Reply:
I don't think of better way than doing this hmmm....

Reply:
Another thought: maybe we should be using the Game Controller API of SDL instead? https://wiki.libsdl.org/SDL3/CategoryGamepad

It seems this API would save us doing a bunch of our own joystick/gamepad abstraction.

Reply:
It would probably be a good idea to integrate gamepad API / controls as a separate PR, and then rebase this port on top of it. That way we have two clean efforts.

Reply:
I'll make a new PR with SDL's Gamepad api